docs/README.md

```diff
@@ -33,6 +33,7 @@ Here follows the list of GitHub Actions topics available in the current document
     - [docker-build-image](#docker-build-image)
     - [docker-dump-containers-logs](#docker-dump-containers-logs)
     - [docker-scan-image-dirs](#docker-scan-image-dirs)
+    - [download-file](#downloade-file)
```

Comment:
```suggestion
    - [download-file](#download-file)
```
(Typo)

Also I would be tempted to make it explicit that this is download from a gh repo --> `github-download-file`? (?) wdyt @alxgomz please?

Reply:
yes indeed, one may think he can download any file from anywehere with the current name 👍